packages/components/datagrid/src/js/datagrid.html

```diff
@@ -66,24 +85,32 @@
                             </oui-datagrid-cell>
                             <oui-skeleton ng-if="row && row.$promise && !$ctrl.hasProperty(row, column.name)"></oui-skeleton>
                         </td>
-                        <td ng-if="$ctrl.hasActionMenu || $ctrl.customizable"
-                            class="oui-datagrid__cell oui-datagrid__cell-sticky">
+                        <td class="oui-datagrid__cell oui-datagrid__cell-sticky"
+                            ng-if="$ctrl.hasActionMenu || $ctrl.customizable">
                             <oui-datagrid-cell ng-if="row && $ctrl.hasActionMenu"
                                 row="row"
                                 column="$ctrl.actionColumn"
                                 index="rowIndex">
                             </oui-datagrid-cell>
                         </td>
                     </tr>
+                    <tr class="oui-datagrid__row"
+                        ng-repeat-end>
+                        <td class="oui-datagrid__row-detail-container"
```

Comment:
`oui-datagrid__row-detail-container`, `oui-datagrid__row-detail`.
It seems like a sort of Block-Element-Element than just Block-Element.

I think we should create another Block level, like `oui-datagrid-row-detail`. And then have Elements like `oui-datagrid-row-detail__container`.